src/features/bounds/model.ts

```diff
@@ -117,8 +117,8 @@ export function getAbsoluteClientBounds(element: SModelElement, domHelper: DOMHe
     const svgElement = document.getElementById(svgElementId);
     if (svgElement) {
         const rect = svgElement.getBoundingClientRect();
-        x = rect.left + window.scrollX;
-        y = rect.top + window.scrollY;
+        x = rect.left + window.pageXOffset;
+        y = rect.top + window.pageYOffset;
```

Comment:
Shall we use `getWindowScroll()` here, too?

Reply:
Done